CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Unreleased
 -   Added Latgalian (`ltg`). (\#296)
 -   Added draft of covering grammar script. (\#297)
 -   Added Portuguese `.phones` files and re-scraped data. (\#290, \#304)
+-   Added Moksha (`mdf`) and modified the pron selector. (\#295)
```

Comment:
Sorry to nitpick but I think the language should go in this section and the change to the pron selector should go in "Under `wikipron/` side of the changelog.